R/derive_vars_query.R

```diff
@@ -220,7 +243,7 @@ assert_valid_queries <- function(queries, queries_name) {
   # check required columns
   assert_data_frame(
     queries,
-    required_vars = exprs(PREFIX, GRPNAME, SRCVAR, TERMCHAR, TERMNUM)
+    required_vars = exprs(PREFIX, GRPNAME, SRCVAR)
   )
```

Comment:
I would move this to `derive_vars_query()`. At the moment `assert_data_frame()` is called three times for `dataset_queries`.